controllers/servicelabels/abstract.go

```diff
@@ -8,16 +8,16 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
shouldn't it be imported as:
`apierrors "k8s.io/apimachinery/pkg/api/errors"`

Reply:
yes, apierrors has been used across